chef/cookbooks/provisioner/templates/default/autoyast.xml.erb

```diff
@@ -12,6 +12,7 @@
       <final_reboot config:type="boolean">false</final_reboot>
       <halt config:type="boolean">false</halt>
       <second_stage config:type="boolean">true</second_stage>
+      <ntp_sync_time_before_installation><%= @ntp_servers.first %></ntp_sync_time_before_installation>
```

Comment:
@dirkmueller can you add something like `<% unless @ntp_servers.empty? -%>` to protect against an empty list of ntp servers?